.github/FUNDING.yml

```
@@ -0,0 +1,6 @@
+github:
+  sponsor: paulpham157
+patreon: PaulPham157
+ko_fi: paulpham157
+
+custom: ['https://www.paypal.me/hungpham2302']
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)